plugins/backend/backstage-plugin-argo-cd-backend/src/service/argocd.service.ts

```diff
@@ -1,32 +1,89 @@
 import { Config } from '@backstage/config';
-import axios, { AxiosRequestConfig } from 'axios';
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
+import { Logger } from 'winston';
+
+export type findArgoAppResp = {
+  name: string;
+  url: string;
+  appName: Array<string>;
+};
+
+export type SyncResponse = {
+  status: 'Success' | 'Failure';
+  message: string;
+};
 
 export interface ArgoServiceApi {
   getArgoToken: (token: string) => Promise<string>;
   getArgoAppData: (
     baseUrl: string,
     argoInstanceName: string,
     options: {
-      name: string,
+      name: string;
       selector: string;
     },
     argoToken: string,
   ) => Promise<object>;
+  createArgoProject: (
```

Comment:
Could it be possible to modify these methods to accept an object instead of direct arguments instead. That way we can more easily see on the caller side what string is assigned to which variable. Also gives more possibilities to modify these later since changes to the method signature don't introduce a breaking changes

Reply:
I believe I have done what is requested. Let me know if it works for you or if I missed what you were saying. Thanks.